src/commands/codepush/lib/file-utils.ts

```diff
@@ -10,11 +11,11 @@ export function isBinaryOrZip(path: string): boolean {
 
 export async function copyFileToTmpDir(filePath: string): Promise<string> {
   if (!(await pfs.stat(filePath)).isDirectory()) {
-    var outputFolderPath: string = await pfs.mkTempDir("code-push");
-    await pfs.rmDir(outputFilePath)
+    let outputFolderPath: string = await pfs.mkTempDir("code-push");
```

Comment:
You are allowed to use sync versions of file system stuff and skip the promises in the CLI.

Reply:
Got it, will fix in the next PR